docs/related/async-raml-datatypes.mdx

```diff
@@ -0,0 +1,324 @@
+---
+id: async_raml_datatypes
+title: Async + RAML Data Types in AMF
+---
+
+:::info Technical Document
+This is a technical document, this information is intended for experienced users
+:::
+
+Within the Async specification it is possible to use raml data types to define a payload, taking into account that the schema format is specified accordingly.
```

Comment:
"raml" should be "RAML" for consistency (please check everywhere in the PR). Same with "api", "yaml", "json", etc.

Reply:
Thanks for the heads-up, I've replaced all inconsistencies:
- every RAML and API reference is now in uppercase
- every json and yaml reference is now in lowercase